arelle/plugin/validate/ESEF/Util.py

```diff
@@ -27,6 +28,17 @@
 
 _: TypeGetText
 
+YEAR_GROUP = "year"
+DISCLOSURE_SYSTEM_YEAR_PATTERN = re.compile(rf"esef-(?P<{YEAR_GROUP}>20\d\d)")
```

Comment:
Mistake on my part in my suggestion. This also needs to be able to parse the unconsolidated names.

```suggestion
DISCLOSURE_SYSTEM_YEAR_PATTERN = re.compile(rf"esef-(?:unconsolidated-)?(?P<{YEAR_GROUP}>20\d\d)")
```

Reply:
Changed, sorry, I didn't check with the unconsolidated, I tried it before pushing this time